donut-app/source/donutappView.mc

```diff
@@ -46,7 +50,33 @@ class donutappView extends Ui.WatchFace {
 
         // Call the parent onUpdate function to redraw the layout
         View.onUpdate(dc);
+        var calories = caloriesBurned(dc, 12872);
```

Comment:
What is 12872? I know it doesn't have to be perfect, but I want to know what it is